source/compiler/sc.h

```diff
@@ -227,6 +232,7 @@ typedef struct s_symbol {
 #define uENUMFIELD  0x040
 #define uMISSING    0x080
 #define uFORWARD    0x100
+#define uSTATIC     0x200
```

Comment:
There's already `sSTATIC` bit which is used to mark symbols as static. It's stored in the `vclass` member. Is there any particular reason why you decided not to use it and add a your own?

Reply:
what you suggest? i don't think will confuse

Reply:
The information which could be obtained from `sSTATIC` can now be obtained from `uSTATIC` and `vclass` (by using `sLOCAL`, `sGLOBAL` and `sSTATEVAR`).

I was thinking of removing `sSTATIC` and use `sGLOBAL`/`sLOCAL`/`sSTATEVAR` in `vclass`. If both `uSTATIC` and `sLOCAL` is set, it implies what `sSTATIC` previously was.

But this would require a lot of editing and could potentially introduce new bugs.